aiida_nanotech_empa/workflows/gaussian/hf_mp2_workchain.py

```diff
@@ -125,7 +125,6 @@ def hf_small_basis_stable(self):
                         'maxcycle': 128,
                         'conver': 7
```

Comment:
is it intentional to keep this conver : 7?

Reply:
Yes, it's intentional. Initially, I was using conver=7 everywhere to make it more robust and more likely to converge (the default conver=8 sometimes took too much time or didn't converge at all for big systems). However, when running these calculations with G16, it complains about the scf convergence being too loose, so i reverted back to default. In this specific instance, I am just getting the spin solution stability correct (which can be quite a heavy calculation) with a small basis set and a loose precision is preferable.